### PR TITLE
Organization: Add postal code field

### PR DESCRIPTION
**Pull request by manuthecoder**
## Summary of the problem
See #5117

## Describe your changes
Add `postal_code` column to `events`

### CODE REVIEW

**Review by sampoder — DISMISSED** [review]
Works like a charm! Next we'll need to make a one-time job that backfills this field
 
Screenshot 2024-02-05 at 1 37 59 PM

**Review by garyhtou — COMMENTED** [review]
Great! Let's add some validations to the model!

For example, if the country is US, then we know all postal codes will be digits-only (regex: `\A\d{5}(?:[-\s]\d{4})?\Z`)

**Comment by sampoder** [comment]
Good idea, I wonder if there's a gem that does that?

**Comment by sampoder** [comment]
@garyhtou @manuthecoder thoughts on https://github.com/dgilperez/validates_zipcode?

**Comment by garyhtou** [comment]
> @garyhtou @manuthecoder thoughts on [dgilperez/validates_zipcode](https://github.com/dgilperez/validates_zipcode)?

That gem looks good to me!

**Comment by manu-gurudath** [comment]
just fixed some merge conflicts

**Comment by manu-gurudath** [comment]
Finished regex validation!

**Review by sampoder — DISMISSED** [review]
LGTM!

File: app/views/events/_form.html.erb [hunk 1]
```diff
@@ -262,6 +262,11 @@
       <%= form.collection_select :stripe_card_shipping_type, Event.stripe_card_shipping_types, :first, :first %>
     </div>
 
+    <div class="field">
+      <%= form.label :postal_code %>
+      <%= form.text_field :postal_code, placeholder: "12345" %>
```
Comment (garyhtou) [inline app/views/events/_form.html.erb]:
```suggestion
      <%= form.text_field :postal_code, placeholder: "90069" %>
```

**Review by garyhtou — DISMISSED** [review]

**Review by garyhtou — APPROVED** [review]

**Pull request merged**